src/main/java/me/karakelley/tictactoe/UserInterface.java

```diff
@@ -0,0 +1,17 @@
+package me.karakelley.tictactoe;
+
+public class UserInterface {
+  private IO io;
+  private BoardPresenter board;
+
+  public UserInterface(IO io, BoardPresenter board) {
```

Comment:
BoardPresenter instance is named `board`. It's a little misleading. I would name it `boardPresenter`.